### DISCUSSION TITLE
Nav Panel overflow

### DISCUSSION THREAD

**Discussion by blitz-research** [michaeljones/breathe#998] [General]
Hi,

The links in the 'nav panel' of my docs are overflowing the width of the panel, eg:

![overflow](https://github.com/user-attachments/assets/d9ae9d0b-2f0d-44cb-b0b9-46cf21f268f1)

I can expand the panel a bit (I assume), but most of the waste is coming from the use of fully qualified enumerator names. Is it possible to only show the unqualified member names here? Adding the qualifiers seems redundant here anyway, as the scope is made quite clear.

I'm not sure if this is a sphinx issue or a breathe one so I've post to both github discussion boards. Apologies if this is bad form.

Bye the way, the docs are generally looking 100X better than they did, thanks for this great project (and sphinx).

Bye,
Mark

**Comment by jakobandersen** [comment]
This is not something Breathe is directly involved in. Please open an issue (if one doesn't exist already) over at Sphinx.
